Grbl_Esp32/src/WebUI/WebServer.cpp

```diff
@@ -280,10 +280,18 @@ namespace WebUI {
         if ((path.substring(0, 4) == "/SD/")) {
             //remove /SD
             path = path.substring(3);
+            int8_t state = get_sd_state(true);
+            if (state != SDCARD_IDLE) {
```

Comment:
Should be SDState::Idle instead of SDCARD_IDLE .  We changed a lot of #defines into proper class-typed enumerations. For reference, the complete enumeration is in src/SDCard.h :

```
enum class SDState : uint8_t {
    Idle          = 0,
    NotPresent    = 1,
    Busy          = 2,
    BusyPrinting  = 2,
    BusyUploading = 3,
    BusyParsing   = 4,
};
```